src/MRA.Jobs.Api/Application.Contracts/TrainingVacancies/Queries/GetTrainingCategoriesQuery.cs

```diff
@@ -0,0 +1,7 @@
+﻿using MRA.Jobs.Application.Contracts.TrainingVacancies.Responses;
+
+namespace MRA.Jobs.Application.Contracts.TrainingVacancies.Queries;
+public class GetTrainingCategoriesQuery : IRequest<List<TrainingCategoriesResponce>>
+{
+    public bool CheckDate { get; set; }

```

Comment:
Is there any reason to have this property?

Reply:
Yes, we have to get the categories thats trainings are in the planned date. For example, if we have a category, it has 2 trainings, but the trainings are planned for later, we do not have to show this category in the trainings page.